src/integration-test/tests/legal/dashboard/steps/dashboard.ts

```diff
@@ -0,0 +1,21 @@
+import { ClaimSearchPage } from 'integration-test/tests/legal/dashboard/pages/claim-search'
+import { ClaimDetailsPage } from 'integration-test/tests/legal/dashboard/pages/claim-details'
+
+const dashboardSearchPage: ClaimSearchPage = new ClaimSearchPage()
+const dashboardClaimDetailsPage: ClaimDetailsPage = new ClaimDetailsPage()
+
+export class DashboardSteps {
+  // todo, not used
```

Comment:
Please delete, steps object should not have `open` method.

Reply:
done